trajectory_generator:
  publish_rate: 50
  optimize_num: 5
  point_min_x: 1200
  point_max_x: 5000
  point_min_y: 5000
  point_max_y: 1300
  window_min_x: 0.6  #0.68
  window_max_x: -0.5 #-0.68
  window_min_y: 0.0  # 0.3
  window_max_y: 1.1  # 1.3
  window_init_x: 0.5
  window_init_y: 1.0
  window_max_a: 0.5
  window_min_v: 0.1
  window_max_v: 0.15
  window_max_d: 0.125
  
